commerce/domain/tax/service/classes/TaxServiceExtensionResolverSample.cls

```diff
@@ -0,0 +1,36 @@
+// This sample is for the situation when different tax behaviors need to be implemented for different locales.
+// These tax behaviors can be -
+// 1. ResolutionState - EXECUTE_DEFAULT (the default Salesforce Internal Tax Api),
+// 2. ResolutionState - EXECUTE_REGISTERED (extended or overridden implementations via the extension point from the default Salesforce Internal Tax Api)
+// 3. ResolutionState - OFF (In this case, the default Salesforce Internal Tax Api will return an empty response).
+// For Salesforce Internal Tax calculation, please see the corresponding documentation.
+
+// An Extension Provider is a custom apex class which extends or overrides the default Salesforce Internal Tax Api.
+// An Extension Resolver is a custom apex class which selects different resolution states (EXECUTE_DEFAULT, EXECUTE_REGISTERED and OFF) for different locales 
+// to execute respective implementations (Extension Providers or the Default Salesforce Internal Tax Api).
+
+// Your custom apex extension providers and the resolver must be registered with the tax extension point and then the resolver must be registered and mapped to the web store via appropriate setup.
+// You can have as many Extension Providers registered as per your use case and select them in your resolver for different locales.
+// Please follow the corresponding salesforce documentation on how to use locales.
+// For more information related to that, please see the corresponding documentation.
+
+// This must implement the commercestoretax.TaxService class in order to be processed by the tax service flow.
+// and it must implement the CommerceExtension.ResolutionStrategy in order to work as a extension resolver and get the different locales and resolutions.
+public class TaxServiceExtensionResolverSample extends commercestoretax.TaxService implements CommerceExtension.ResolutionStrategy {
+    public CommerceExtension.Resolution resolve() {
+        // The Sample Extension Provider registered with developer name as 'tax_extension_provider_for_us' will be selected for execution for en_US locale
+        if(CommerceExtension.ExtensionInfo.getLocaleString() == 'en_US') {
```

Comment:
Extract to constants or use platform constants in APEX rather than bare String values